packages/arb-token-bridge-ui/src/state/app/state.ts

```diff
@@ -69,6 +69,8 @@ export interface MergedTransaction {
 }
 
 export interface TeleporterMergedTransaction extends MergedTransaction {
+  // note: in contrast to general deposits which use `parentToChildMsgData`,
+  // Teleport transfers still follow L1/L2/L3 terminology, so we have `l1ToL2MsgData` and `l2ToL3MsgData`
```

Comment:
If we use `/** Comment here */` rather than inline comment, it shows on hover:
![image](https://github.com/user-attachments/assets/1eb1ed1f-9a6e-4e70-86d9-c1b89576efe4)